api/clients/accountant.go

```diff
@@ -8,15 +8,15 @@ import (
 	"sync"
 	"time"
 
-	commonpb "github.com/Layr-Labs/eigenda/api/grpc/common"
+	disperser_rpc "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
 	"github.com/Layr-Labs/eigenda/core"
 	"github.com/Layr-Labs/eigenda/core/meterer"
 )
 
 var requiredQuorums = []uint8{0, 1}
 
 type Accountant interface {
-	AccountBlob(ctx context.Context, numSymbols uint64, quorums []uint8) (*commonpb.PaymentHeader, error)
+	AccountBlob(ctx context.Context, numSymbols uint64, quorums []uint8) (*core.PaymentMetadata, error)
 }
```

Comment:
Makes sense. Can update the names from `Accountant-> IAccountant` for interface and `accountant-> Accountant`

Reply:
Think `Accounter` would be more golangy? But in either case, Unless this interface is already used in many places, best practice is to just delete it. Interfaces should typically be defined at the call site, not where the struct is defined. That's because different call sites need different subsets of methods of a struct typically, so best to define the minimal interface needed (makes it easier to write a fake for that minimal interface).

Reply:
I removed the interface definition and renamed the struct to Accounter

Reply:
Wah sorry! The struct should still be called Accountant. I was saying IF we were to have an interface here (which I don't think we should), then it could be called `Accounter` instead of `IAccountant`. It's typical of golang to use verbs for interface, like Writer and Reader.

Reply:
LGTM after [6643ca9](https://github.com/Layr-Labs/eigenda/pull/928/commits/6643ca9280c1c4d8cd7c59cde05a939e9813a5eb); Ill resolve this